### DISCUSSION TITLE
Allow content scrolling when sheet is not fully expanded

### DISCUSSION THREAD

**Discussion by qinyiyao** [gorhom/react-native-bottom-sheet#1276] [Ideas]
For example, this sheet in iOS is not fully expanded but you can still scroll within it if there’s an existing offset. If you scroll back to the top in this state then scroll again, the sheet expands. The scroll area always fits within visible area.

Another interesting use case is allow the content to be scrolled without expanding the sheet even if there’s no current offset. The user would have to pull up the handle manually to enlarge the scroll area.

![RPReplay_Final1674550156 2023-01-24 00_53_22](https://user-images.githubusercontent.com/6311717/214249058-03dc894d-7600-4f36-909e-8d02f0b44494.gif)

**Comment by palok** [comment]
I second this feature request 👍

**Comment by jaakola-aleksanteri** [comment]
Any news about this feature request?
